AssetManagementTestProject/TestCases/LoginTest.cs

```diff
@@ -7,27 +7,49 @@ namespace AssetManagementTestProject;
 [TestFixture]
 public class LoginTest : NUnitWebTestSetup
 {
+    protected ChangePassword1stTimePage ChangePw1stTime;
+
+    [SetUp]
+    public void SetUp()
+    {
+        ChangePw1stTime = new ChangePassword1stTimePage();
+    }
     [Test]
     public void UserLoginSuccess()
     {
-        LoginPage.Login(Constant.ADMIN_USERNAME_HN, Constant.BASE_ADMIN_PASSWORD);
-        Asserter.AssertStringEquals(HomePage.ReturnHomePageUrl(), Constant.BASE_URL);
+        LoginPage.Login(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN);
+        DriverBaseAction.WaitToBeVisible(HomePage.ReturnHeaderHomePage());
+        DriverBaseAction.CompareUrls(Constant.BASE_URL);
     }
     [Test]
     public void UserLoginFirstTime()
     {
-        LoginPage.Login(Constant.STAFF_USERNAME_1, Constant.BASE_STAFF_PASSWORD_1);
+        LoginPage.Login(Constant.STAFF_USERNAME_1ST_TIME, Constant.STAFF_PASSWORD_1ST_TIME);
```

Comment:
How the STAFF_USERNAME_1ST_TIME user and STAFF_USERNAME user can exist in the default database?

Reply:
You should create default data by sending the API and clear data after the test is done

Reply:
I dont think our group's API is ready for testing.
![image](https://user-images.githubusercontent.com/79431746/205424892-75441c3a-c2f2-4561-8590-c93766cb2906.png)

And when I sent a GET request to the base url I got a html response body (We have yet to learn how to send text/html request body) so I'm kinda stuck at this. I'm going to update if I find something useful
![image](https://user-images.githubusercontent.com/79431746/205424984-cf90580d-72cc-4b51-a37b-cda4781309b4.png)

Reply:
Done with the creating part, I will update this thread when I finish with clearing data by disabling created user